README.md

```diff
@@ -106,6 +108,9 @@ Metadata has the following attributes:
   * **commits_since_annotated_tag** *Optional* number of commits since the last annotated tag.
 * **source_location** *Optional* path and line number of the source file that was recorded. Most commonly, this is the test case that was recorded. Format is `<path>:<line>`, with `line` being optional.
 * **test_status** *Optional* status of the test that ran to generate the AppMap. Valid values are `succeeded` or `failed`.
+* **test_failure** *Optional* Details about a failed test.
```

Comment:
The example below is now missing quite a few of the attributes. Might be worth updating it at some point....